### ISSUE TITLE
Unable to compile Align-It with openbabel at the end of make

### ISSUE THREAD

**Issue by gryffNx** [openbabel/openbabel#2667]
Good afternoon,

I am working on building and compiling Align-It for our research informatics team with the company I work for, and running into some errors on a Linux 2023 AMI instance. We have successfully installed Shape-It on a Rocky Linux server before, and the RI team wanted to utilize Align-It as well, and looks like it should be a similar compile and build process, so not sure if there's something different among my Linux dev box, versus the Rocky Linux box. That would have to be tested next week.. I am hoping to gain any insight on the 'undefined reference' errors I am receiving at the end of the 'make' command for my align-it project.

/home/ec2-user/openbabel/include/openbabel/math/vector3.h:274:10: note: declared here
274 | bool operator== ( const vector3& ) const;
| ^~~~~~~~
[100%] Linking CXX executable align-it

but then started going into errors like this:

/usr/bin/ld: CMakeFiles/align-it.dir/src/result.cpp.o: in function Result::Result()': result.cpp:(.text+0x170): undefined reference to OpenBabel::OBMol::OBMol()'
/usr/bin/ld: CMakeFiles/align-it.dir/src/utilities.cpp.o: in function positionMolecule(OpenBabel::OBMol*, SiMath::Matrix&, SolutionInfo&)': utilities.cpp:(.text+0x1920): undefined reference to OpenBabel::OBMol::BeginAtom(__gnu_cxx::__normal_iterator<OpenBabel::OBAtom**, std::vector<OpenBabel::OBAtom*, std::allocatorOpenBabel::OBAtom* > >&)'
/usr/bin/ld: utilities.cpp:(.text+0x1b93): undefined reference to OpenBabel::OBAtom::SetVector(double, double, double)' /usr/bin/ld: utilities.cpp:(.text+0x1ba0): undefined reference to OpenBabel::OBMol::NextAtom(__gnu_cxx::__normal_iterator<OpenBabel::OBAtom**, std::vector<OpenBabel::OBAtom*, std::allocatorOpenBabel::OBAtom* > >&)'
/usr/bin/ld: CMakeFiles/align-it.dir/src/utilities.cpp.o: in function TransformMolecule(OpenBabel::OBMol*, SiMath::Matrix&, Coordinate&, Coordinate&)': utilities.cpp:(.text+0x1e74): undefined reference to OpenBabel::OBMol::BeginAtom(__gnu_cxx::__normal_iterator<OpenBabel::OBAtom**, std::vector<OpenBabel::OBAtom*, std::allocatorOpenBabel::OBAtom* > >&)'
/usr/bin/ld: utilities.cpp:(.text+0x1f94): undefined reference to OpenBabel::OBAtom::SetVector(double, double, double)' /usr/bin/ld: utilities.cpp:(.text+0x1fa1): undefined reference to OpenBabel::OBMol::NextAtom(__gnu_cxx::__normal_iterator<OpenBabel::OBAtom**, std::vector<OpenBabel::OBAtom*, std::allocatorOpenBabel::OBAtom* > >&)'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/align-it.dir/build.make:593: align-it] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/align-it.dir/all] Error 2
make: *** [Makefile:156: all] Error 2

Thank you in advance for any help! I have been searching for possible solutions online but have not found anything helpful. I believe I have all the necessary components installed, but unsure if there's something incorrect or missing from my CMakeLists.txt file.

**Comment by ghutchis** [comment]
Okay, but Align-It isn't part of Open Babel. It seems as if it's not linking correctly in their CMakeLists.txt. I'd suggest submitting a bug report to them.

**Comment by gryffNx** [comment]
Yeah I was hoping if anybody in this community would have any insight or familiarity with Align-It. Doesn't look like there's been any recent updates about Align-It on the github page.
